### DISCUSSION TITLE
Content Model

### DISCUSSION THREAD

**Discussion by pglevy** [Bixal/methods#103] [Ideas]
Hello! What do you think about including some method about building content models? I think mostly Russ has the corner on this task - he made one for FSIS, Bixal.com, and DEA. Its a nice tool to do before sitemapping so you can think about the concepts and data before deciding on page/site structure. (Submitted by: katharine.carr@bixal.com)

**Comment by pglevy** [comment]
@katharine-carr, can you attach some of the samples you're thinking of here for reference?

**Comment by pglevy** [comment]
Here's an [article about _concept modeling_](https://articles.uie.com/concept_models/) I was thinking about for a previous issue.

**Comment by katharine-carr** [comment]
Here are some links to work that Russ has done (he said it was ok to share!): 

DEA Museum content model: [https://coggle.it/diagram/X8aZiiM6PVtkKa8-/t/designing-the-web-component-of-the-in-person-interactives/2a57f42ae2a5e00ce54f1f561312ef58615121944d7cd348af03be8baaf0dda6](url)

Bixal.com: https://coggle.it/diagram/Xs0qEPYr3Jsgt9ar/t/%E2%9A%99%EF%B8%8F-case-study-a-case-study-without-let-bixal-worry-for-me/56e57eef9eae22e4d7ac8621c5ce9d097117d36943d7ce4b31779005a45e92ab

Here's an article from A List Apart that is talking about what I am thinking of: https://alistapart.com/article/content-modelling-a-master-skill/

And this one, you can scroll straight to the paragraph called 'Modeling Content': https://uxpamagazine.org/designing-future-friendly-content/

**Comment by pglevy** [comment]
Moving this to Discussions for now because I think there's more to dig into here about terminology and what we want to accomplish with each type of method/artifact.
